server/resources/resource_management_test.go

```diff
@@ -1252,6 +1259,66 @@ var _ = Describe("Resource manager", func() {
 				Expect(ok).To(BeTrue())
 			})
 		})
+
+		Describe("When tenant_id is blacklisted", func() {
+			BeforeEach(func() {
+				schemaID = "blacklisted_tenant_id"
+			})
+
+			Context("A member", func() {
+				BeforeEach(func() {
+					auth = memberAuth
+				})
+
+				It("Should create own resource when tenant_id isn't provided and not fill it in",
+					func() {
+						err := resources.CreateResource(
+							context, testDB, fakeIdentity, currentSchema, map[string]interface{}{
+								"id":        memberResourceID,
+								"domain_id": domainAID,
+							})
+
+						Expect(err).NotTo(HaveOccurred())
+
+						result := context["response"].(map[string]interface{})
+						resource, ok := result[schemaID]
+						Expect(ok).To(BeTrue())
+						Expect(resource).To(HaveKeyWithValue("id", memberResourceID))
+						Expect(resource).To(HaveKeyWithValue("domain_id", domainAID))
+						Expect(resource).NotTo(HaveKeyWithValue("tenant_id", auth.TenantID()))
+					})
+
+				It("Should not create resource when blacklisted tenant_id is provided", func() {
+					err := resources.CreateResource(
+						context, testDB, fakeIdentity, currentSchema, map[string]interface{}{
+							"id":        memberResourceID,
+							"tenant_id": memberTenantID,
+							"domain_id": domainAID,
+						})
+
+					Expect(err).To(HaveOccurred())
+
+					resErr, ok := err.(resources.ResourceError)
+					Expect(ok).To(BeTrue())
+					Expect(resErr.Problem).To(Equal(resources.Unauthorized))
```

Comment:
shouldn't it be a bad request?

Reply:
This test is simply documenting already existing behavior.

Changing this is trivial but results in 2 failures in `server/server_test.go`. If changing those expectations is fine, I'll do it. Waiting for go-ahead.

Failing tests:
server_test.go:553:"Should not create exposed with incorrect fields"
server_test.go:555:"Should not create forbidden with incorrect fields"

Reply:
So I guess this is not that important. If it's still desired I can create another PR with this simple change.